kukur/base.py

```diff
@@ -43,6 +43,20 @@ def to_data(self) -> dict[str, Any]:
         """Convert to JSON object."""
         return dict(source=self.source, tags=self.tags, field=self.field)
 
+    def get_series_name(self) -> str:
+        """Get the series name with tags and fields included.
+
+        For sources that cannot handle tags and fiels yet."""
+        series_string = ""
+        for tag_key, tag_value in self.tags.items():
+            if tag_key == "series name":
+                series_string = series_string + tag_value
+                continue
+            series_string = series_string + "," + f"{tag_key}={tag_value}"
```

Comment:
This will break when there is no "series name".
Better `join` on a list and add the series name at the start if it exists.